packages/core/src/Stores/portfolio-store.js

```diff
@@ -103,7 +103,6 @@ export default class PortfolioStore extends BaseStore {
         if (this.has_subscribed_to_poc_and_transaction) {
             this.clearTable();
         }
-        this.is_loading = true;
```

Comment:
This line is no longer needed because the initial `is_loading` state is changed to `true` in this store.
It should be `true` initially since when the page first loads, there is no data fetched yet, and we always proceed with fetching which should make loader visible.